app/init.php

```diff
@@ -927,6 +927,18 @@ function (mixed $value) {
         'hostname' => $request->getHostname(),
     ], Document::SET_TYPE_APPEND);
 
+    $hostnames = explode(',', App::getEnv('_APP_CONSOLE_HOSTNAMES', ''));
+    if (is_array($hostnames)) {
```

Comment:
Hmm if `App::getEnv('_APP_CONSOLE_HOSTNAMES', '')` and an empty string, `$hostnames` will be an array with 1 empty string. That shouldn't be added as a platform, right?

Reply:
ohhh, one of those weird PHP things. We should fix it before releasing.